ultralytics/nn/modules/head.py

```diff
@@ -179,13 +179,14 @@ def __init__(self, nc=80, kpt_shape=(17, 3), ch=()):
     def forward(self, x):
         """Perform forward pass through YOLO model and return predictions."""
         bs = x[0].shape[0]  # batch size
-        if self.export:
-            kpt = torch.cat([torch.permute(self.cv4[i](x[i]), (0, 2, 3, 1)).reshape(bs, -1, self.nk) for i in range(self.nl)], 1)
-        else:
-            kpt = torch.cat([self.cv4[i](x[i]).view(bs, self.nk, -1) for i in range(self.nl)], -1)  # (bs, 17*3, h*w)
+        kpt = torch.cat([self.cv4[i](x[i]).view(bs, self.nk, -1) for i in range(self.nl)], -1)  # (bs, 17*3, h*w)
```

Comment:
why have you changed Detect forward?

Reply:
This change is in Pose forward, and it was part of the same experiment with pose models mentioned above. I will change them back.